simpleflow/swf/executor.py

```diff
@@ -570,14 +570,46 @@ def resume(self, a_task, *args, **kwargs):
 
         return future
 
+    def _compute_priority(self, priority_set_on_submit, a_task):
+        """
+        Computes the correct task priority, with the following precedence (first
+        is better/preferred):
+        - priority set with self.submit(..., __priority=<N>)
+            (encoded as False if wasn't set)
```

Comment:
Oh yeah! I was thinking of having constants mapping to strings, but I thought it was ugly and prone to bad comparisons. I like that, thanks :)